test/processors/concerns/http_files_test.rb

```diff
@@ -46,7 +46,7 @@ def logger
 
   it 'can download from https url' do
     stub_request(:get, https_uri.to_s).
-      with(:headers => { 'Host' => 'test.prx.org:443' } ).
+      with(:headers => { 'Host' => 'test.prx.org' } ).
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.